.env.template

```diff
@@ -1,11 +0,0 @@
-MONGODB_URI='mongodb://localhost:27017/quiz-game'
```

Comment:
Please restore this file

Reply:
I changed name from .env.template to just .env. Should i return word .template to this file?